src/rgw/driver/d4n/d4n_directory.cc

```diff
@@ -178,7 +178,7 @@ int ObjectDirectory::get(const DoutPrefixProvider* dpp, CacheObj* object, option
     object->objName = std::get<0>(resp).value()[0];
     object->bucketName = std::get<0>(resp).value()[1];
     object->creationTime = std::get<0>(resp).value()[2];
-    object->dirty = std::stoi(std::get<0>(resp).value()[3]);
```

Comment:
You must modify the commit message to remove support for 'simple' get request, as get requests for all scenarios work even when an instance is supplied.